1.7/administration/high-availability/index.md

```diff
@@ -0,0 +1,29 @@
+---
+post_title: High-Availibility 
+menu_order: 3.1
+---
+
+DC/OS supports high-availability configurations, including mult-zone, multi-region, and hybrid cloud. 
+
+# Terminology
+
+## Rack
+A rack is typically composed of a set of servers (nodes) with its own power supply and switch (or switches), all attached to the same frame. On public cloud platforms such as AWS, there is no equivalent concept of a rack.
+
+## Region
```

Comment:
I would define this after "zone" definition, because this talks about zone without defining it.